tests/dtm/test_single_process_restart.py

```diff
@@ -1036,3 +1036,75 @@ def test_copy_object_during_rgw_restart(self):
             assert_utils.assert_true(resp, "Checksum validation Failed.")
         self.test_completed = True
         self.log.info("ENDED: Verify copy object during rgw restart using pkill.")
+
+    @pytest.mark.lc
+    @pytest.mark.dtm
+    @pytest.mark.tags("TEST-41245")
+    def test_multiple_rgw_single_m0d_restart(self):
+        """Verify IO operations work during multiple rgw restarts, single m0d restart
+        and multiple rgw restarts."""
+        self.log.info("STARTED: Verify IO operations work during multiple rgw restarts,"
+                      " single m0d restart and multiple rgw restarts.")
+        bucket_name = 'bucket-test-41231'
+        object_prefix = 'object-test-41231'
+        log_file_prefix = 'test-41231'
+        que = multiprocessing.Queue()
+        num_loops = num_buckets = self.test_cfg["test_41245"]["num_loop"]
+        rgw_restarts = self.test_cfg["test_41245"]["rgw_restarts"]
+
+        self.log.info("Step 1: Create %s buckets for IO operation during restarts", num_buckets)
+        resp = self.s3_test_obj.create_multiple_buckets(num_buckets, 'test-41245')
+        bucket_list = resp[1]
+        self.log.info("Step 1: Bucket created in healthy mode ")
+
+        self.log.info("Step 2: Perform Writes,Reads and Validate operation for %s iterations:",
+                      num_loops)
+        proc_ios = multiprocessing.Process(target=self.dtm_obj.perform_write_op, args=(
+            bucket_name, object_prefix, self.test_cfg['clients'], self.test_cfg['samples'],
+            log_file_prefix, que, self.test_cfg['size'], num_loops, bucket_list))
+        proc_ios.start()
+
+        self.log.info("Step 3 : Perform RGW Process Restarts for %s iteration during IO operations",
+                      rgw_restarts)
+        resp = self.dtm_obj.process_restart(master_node=self.master_node_list[0],
```

Comment:
don't u want to start this one as well in background? or is it rgw restart and then m0d? I thought together.